app/build.gradle

```diff
@@ -9,10 +9,6 @@ androidExtensions {
     experimental = true
 }
 
-kapt {
-    generateStubs = true
-}
```

Comment:
By removing this line, what changes?

Reply:
This annotation processor API was deprecated after the the kotlin version 1.0.4. Just add the `apply plugin: 'kotlin-kapt' ` to use the annotations from the newer version